requirements/nominal_test.txt

```diff
@@ -0,0 +1 @@
+dython
```

Comment:
any min version?

Reply:
I think not necessary here.

Reply:
```suggestion
dython >=0.7.0
```

Reply:
See for `oldest`:

```
ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
The conflict is caused by:
    The user requested numpy==1.17.2
    torch 1.8.1 depends on numpy
    torchvision 0.9.1 depends on numpy
    rouge-score 0.0.4 depends on numpy
    transformers 4.4.0 depends on numpy>=1.17
    sacrebleu 2.0.0 depends on numpy>=1.17
    mir-eval 0.6 depends on numpy>=1.7.0
    fast-bss-eval 0.1.0 depends on numpy
    dython 0.7.0 depends on numpy>=1.19.5
 ```

Reply:
strange, but reverted to unblock this PR :otter: